### DISCUSSION TITLE
https support for secure access to tools page

### DISCUSSION THREAD

**Discussion by Chriwro** [mcguirepr89/BirdNET-Pi#1014] [Ideas]
Unfortunately there is no https support build in birdnetpi. This means, almost everyone could do a man in the middle attack if you type in your password for the tools page. With that an attacker has access to the web terminal with which he could run his own code. An implementation of  certbot could solve this problem easily.

**Comment by Svardsten53** [comment]
I think almost everyone who has their installations accessible from the internet has solved this themselves. Caddy supports https including certificate management and it is easy to specify this in the path in the configuration.
